lambdas/tests/unit/handlers/test_create_document_reference_handler.py

```diff
@@ -92,11 +92,11 @@ def test_lambda_handler_missing_environment_variables_type_lg_returns_400(
 
 @pytest.mark.parametrize("environment_variable", arf_environment_variables)
 def test_lambda_handler_missing_environment_variables_type_arf_returns_500(
-    set_env,
-    monkeypatch,
-    arf_type_event,
-    environment_variable,
-    context,
+        set_env,
+        monkeypatch,
+        arf_type_event,
+        environment_variable,
+        context,
```

Comment:
Did you run `make format` before pushing? These changes to the format look a bit strange

Reply:
I guess this is my IDE formatting it

When I do make format, I get 49 files changed:
![image](https://github.com/nhsconnect/national-document-repository/assets/127327856/715cbde8-c3ff-4abf-8e42-f4efa841e4be)